build.gradle

```diff
@@ -179,8 +184,14 @@ task buildAwsLc {
                 args "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
 
                 if (isFips) {
+                    println "Building AWS-LC in FIPS mode"
                     args '-DFIPS=1'
                 }
+
+                if (allowFipsTestBreak) {
+                    println "Building AWS-LC with hooks to break FIPS tests"
+                    args '-DFIPS_BREAK_TEST=TESTS'
```

Comment:
[LC appends](https://github.com/aws/aws-lc/blob/main/CMakeLists.txt#L592-L594) the value of `FIPS_BREAK_TEST` to `BORINGSSL_FIPS_BREAK_` and defines new flag. In this case, it would result to `BORINGSSL_FIPS_BREAK_TESTS`. I'm guessing the original intention was to have separate builds for each test-breaking scenario, but it later got dropped in favor of a single breakable build and control the scenario via environment variable. There is one exception and that is `BORINGSSL_FIPS_BREAK_FFC_DH` that is used to break some key derivation test: it requires a different build by setting `FIPS_BREAL_TEST` to `FFC_DH` and cannot be triggered via environment variable.